### ISSUE TITLE
Game doesn't load on Pixel 3xl

### ISSUE THREAD

**Issue by bobums** [yairm210/Unciv#3643]
I have played this game for a long time. Stopped playing for about a month and tried to load it one day and it shows the language screen for about a nano second and then just goes to a blue screen and sits there.  I have cleared cache, force stop, uninstalled and reinstalled and still the same. I am on a google pixel 3xl.

**Comment by rallion** [comment]
I have the same problem. It looks like it happens if the Android Display Size is set to Small.

**Comment by jahodovykolac** [comment]
I have a different problem. When I click on a saved game and tap Load game, a popup saying "Loading game" shows up for two seconds and then closes. I can do anything like I never tried to load the game. I can't remember now, but I think I have to click the game again, it starts loading info (like difficulty, name etc.) and then it loads without any pop-ups.

**Comment by yairm210** [comment]
@rallion What device are you on? And what version? (number, please)

**Comment by rallion** [comment]
@yairm210 This is on a Pixel 3 XL, Android 11 with security patches up to Feb 5 2021.

Some extra details regarding how the game seems to interact with that setting on my device: Only the "Small" setting always causes the game to always show the blue screen. Changing the setting (from anything to anything) while the game is running also causes the display to just show as solid blue when I switch back to it, but as long as it isn't set to "small," it will display properly if I restart the app.

**Comment by yairm210** [comment]
How extremely odd. Does this happen on the world screen as well as the main screen? And what do you mean by "small" setting, is this a phone feature of some sort?

**Comment by rallion** [comment]
Yeah, it seems to happen on any screen.

There are two scaling settings in the phone's Display settings: Font Size just scales text, and Display Size scales lots of other things too. I just checked, and it seems that changing either of these requires the game to be restarted for the display to work, but setting Display Size to "small" (other settings are Default, Large, Larger, Largest) is still the only thing I've found that seems to break the display even on a fresh app start.
